scripts/hoist-internals.js

```diff
@@ -15,7 +15,7 @@ log.addLevel('success', 3001, { fg: 'green', bold: true });
 log.info(prefix, 'Hoisting internal packages');
 
 const getLernaPackages = () =>
-  fse.readJson(path.join(__dirname, '..', 'lerna.json')).then(json => json.packages);
+  fse.readJson(path.join(__dirname, '..', 'package.json')).then(json => json.workspaces);
```

Comment:
I've noticed that this script always yields `Hoisted 0 packages`. So either something is broken, or `yarn` does what's needed. @ndelangen can you please check?

Reply:
I think yarn workspaces already hoists everything?

Reply:
Looks like that, I just want to be sure about that

Reply:
This entire script can be removed if this PR comes thru